graphql/tests/myPublicKeys/data.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
is this intentionally blank?

Reply:
yeah the test suite assumes there will always be a data.json file so we have to declare an empty file.